examples/oci/deploy-all.sh

```diff
@@ -33,19 +35,35 @@ if [[ "$(basename $PWD)" != "oci" ]]; then
   fi
 fi
 
-# Deploy Jaeger (All-in-One with memory storage)
-echo "🟣 Step 1: Installing Jaeger..."
-helm upgrade --install --force jaeger ./helm-charts/charts/jaeger \
-  --set provisionDataStore.cassandra=false \
-  --set allInOne.enabled=true \
-  --set storage.type=memory \
-  --set-file userconfig="./config.yaml" \
-  -f ./jaeger-values.yaml
+if [[ "$MODE" == "upgrade" ]]; then
+  echo "🟣 Upgrade mode: Upgrading Jaeger and Prometheus..."
+  helm upgrade --install --force jaeger ./helm-charts/charts/jaeger \
+    --set provisionDataStore.cassandra=false \
+    --set allInOne.enabled=true \
+    --set storage.type=memory \
+    --set-file userconfig="./config.yaml" \
+    --set-file uiconfig="./ui-config.json" \
+    -f ./jaeger-values.yaml
+
+  kubectl apply -f prometheus-svc.yaml
+  helm upgrade --install --force prometheus -f monitoring-values.yaml prometheus-community/kube-prometheus-stack
+else
+  echo "🟣 Clean mode: Uninstalling and reinstalling Jaeger and Prometheus..."
+  helm uninstall jaeger --ignore-not-found || true
+  helm uninstall prometheus --ignore-not-found || true
+  sleep 5
```

Comment:
why sleep? Can you not check in a loop via kubectl that the resource was destroyed?